scripts/controlnet.py

```diff
@@ -230,7 +250,20 @@ def uigroup(self, is_img2img):
 
         ctrls += (enabled,)
         # infotext_fields.append((enabled, "ControlNet Enabled"))
-            
+        
+        def send_dimensions(image):
+            def closesteight(num):
+                rem = num % 8
+                if rem <= 4:
+                    return round(num - rem)
+                else:
+                    return round(num + (8 - rem))
+            if(image):
+                interm = np.asarray(image.get('image'))
+                return gr.Slider.update(value=closesteight(interm.shape[1])), gr.Slider.update(value=closesteight(interm.shape[0]))
```

Comment:
```suggestion
                return closesteight(interm.shape[1]), closesteight(interm.shape[0])
```
[update()](https://www.gradio.app/docs/#update-header) is only necessary to set properties other than value.